### ISSUE TITLE
moby-derp should start stopped containers

### ISSUE THREAD

**Issue by Supermathie** [mpalmer/moby-derp#9]
I believe moby-derp should start any stopped managed containers when run.

In our use case, after a reboot docker might start the sidecar before the pod root which will fail since the root isn't running.

This situation can be replicated with:

```
michael@andor:~/moby-derp$ RUBYLIB=~/prog/moby-derp/ bundle exec moby-derp redis.yml
Checking root container
Deleting container redis (724fe0b8a5e9) because it is out-of-date
Creating new container redis
Checking container redis.server
Deleting container redis.server (a3d3cf3aafd4) because it is out-of-date
Creating new container redis.server

michael@andor:~/moby-derp$ docker stop redis.server 
redis.server

michael@andor:~/moby-derp$ RUBYLIB=~/prog/moby-derp/ bundle exec moby-derp redis.yml
Checking root container
Container redis is up-to-date
Checking container redis.server
Container redis.server is up-to-date
```

And redis.server is still not running.

Tested against moby-derp at fe8bb342.

**Comment by mpalmer** [comment]
I see where you're coming from here.  Is limiting it to `restart: always` containers sufficient for your use-case?  Poking stopped containers that aren't supposed to always be running would be a bit rude, and trying to reimplement the logic around `on-failure` and `unless-stopped` (and trying to figure out from Docker whether those are true) would be a lot more complicated than (more-or-less) just `container.kick_in_pants if container.restart == "always" && container.not_running`.

**Comment by Supermathie** [comment]
> Is limiting it to restart: always containers sufficient for your use-case?

That sounds sensible! That'd do for us.

**Comment by SamSaffron** [comment]
PR here: 

https://github.com/mpalmer/moby-derp/pull/17

**Comment by mpalmer** [comment]
Closed by #17.
